AdSec/.NET/AdsToApi/AdsToApi.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using Oasys.AdSec;
+using Oasys.AdSec.IO.Serialization;
+using Oasys.Collections;
+
+namespace AdsToApi
+{
+    /// <summary>
+    /// This example shows how to convert an .ads file into API objects.
+    ///
+    /// You might like to run the 'ApiVersion' example first, just to check
+    /// that the API is installed correctly.
+    /// </summary>
+    public static class AdsToApi
```

Comment:
Maybe we should call this project/class something like `OpenAdSecFile`? (I'm not sure if users know what "Ads" is?)